packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -0,0 +1,292 @@
+  const {
+    resultData,
+    hideUnit,
+    isTruncated,
+    hasResultError,
+    hasResultNoData,
+    intl,
+    ...customProps
+  } = props;
+
+  let clinicalResultBloodPressureDisplay = null;
+
+  if (hasResultError) {
+    clinicalResultBloodPressureDisplay = <ResultError />;
+  } else if (hasResultNoData) {
+    clinicalResultBloodPressureDisplay = <NoData />;
+  } else {
+    const CompareTemplate = (s = null, d = null) => ({
+      systolic: s,
+      diastolic: d,
+    });
+    const compareUnits = new CompareTemplate();
+    const compareConceptDisplays = new CompareTemplate();
+    const compareDatetimeDisplays = new CompareTemplate();
+    const compareStatusIsInError = new CompareTemplate();
+
+    let hasModifiedIcon = false;
+    let hasCommentIcon = false;
+    let hasUnverifiedIcon = false;
+
+    let iconGroupDisplayElement = null;
+    let conceptDisplayElement = null;
+    let datetimeDisplayElement = null;
+    const decoratedResultDisplay = [];
+
+    const hasSystolic = resultData.systolic;
+    const noDataSystolic = (hasSystolic && resultData.systolic.resultNoData === true);
+    if (hasSystolic && !noDataSystolic) {
+      const {
+        result = null,
+        status = null,
+        conceptDisplay = null,
+        datetimeDisplay = null,
+        isModified = null,
+        hasComment = null,
+        isUnverified = null,
+      } = hasSystolic;
+
+      if (!isEmpty(result.unit)) { compareUnits.systolic = result.unit.trim().toLowerCase(); }
+      if (!isEmpty(status)) { compareStatusIsInError.systolic = checkIsStatusInError(hasSystolic); }
+      if (!isEmpty(conceptDisplay)) { compareConceptDisplays.systolic = conceptDisplay.trim().toLowerCase(); }
+      if (!isEmpty(datetimeDisplay)) { compareDatetimeDisplays.systolic = datetimeDisplay.trim().toLowerCase(); }
+      if (!isEmpty(isModified)) { hasModifiedIcon = isModified; }
+      if (!isEmpty(hasComment)) { hasCommentIcon = hasComment; }
+      if (!isEmpty(isUnverified)) { hasUnverifiedIcon = isUnverified; }
+    }
+
+    const hasDiastolic = resultData.diastolic;
+    const noDataDiastolic = (hasDiastolic && resultData.diastolic.resultNoData === true);
+    if (hasDiastolic && !noDataDiastolic) {
+      const {
+        result = null,
+        status = null,
+        conceptDisplay = null,
+        datetimeDisplay = null,
+        isModified = null,
+        hasComment = null,
+        isUnverified = null,
+      } = hasDiastolic;
+
+      if (!isEmpty(result.unit)) { compareUnits.diastolic = result.unit.trim().toLowerCase(); }
+      if (!isEmpty(status)) { compareStatusIsInError.diastolic = checkIsStatusInError(hasDiastolic); }
+      if (!isEmpty(conceptDisplay)) { compareConceptDisplays.diastolic = conceptDisplay.trim().toLowerCase(); }
+      if (!isEmpty(datetimeDisplay)) { compareDatetimeDisplays.diastolic = datetimeDisplay.trim().toLowerCase(); }
+      if (!isEmpty(isModified)) { hasModifiedIcon = isModified; }
```

Comment:
Arn't `isModified`, `hasComment `, and `isUnverified` booleans? If yes, I don't think we need to check isEmpty since thats a string helper. If these aren't booleans, can we rename these so they don't represent a bool?